hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoGetAccountBalanceHandler.java

```diff
@@ -82,19 +86,47 @@ public void validate(@NonNull final QueryContext context) throws PreCheckExcepti
         final var query = context.query();
         final var accountStore = context.createStore(ReadableAccountStore.class);
         final CryptoGetAccountBalanceQuery op = query.cryptogetAccountBalanceOrThrow();
+        mustExist(op.balanceSource(), INVALID_CONTRACT_ID);
```

Comment:
Is this error code correct? `op.balanceSource()` can be `ACCOUNT_ID` or `CONTRACT_ID`, so it seems misleading to always use `INVALID_CONTRACT_ID`. Especially since this endpoint is called _CryptoGetACCOUNTBalanceHandler_. I would expect we default to `INVALID_ACCOUNT_ID` like in the `UNSET` case just below.

EDIT: `validateContractId(...)` and `validateAccountId(...)` both call `mustExist(...)`, so I think we can remove this line here